packages/3dt/build

```diff
@@ -24,7 +24,7 @@ mkdir -p $(dirname $slave_public_service)
 
 cat <<EOF | tee $slave_service $slave_public_service
 [Unit]
-Description=Diagnostics: DC/OS Distributed Diagnostics Tool
+Description=DC/OS Diagnostics (3DT) Agent: exposes component health
```

Comment:
3DT only exposes component health. It just groups them component health by node when it aggregates on the masters.